src/main/java/bio/terra/service/snapshot/flight/export/SnapshotExportWriteManifestStep.java

```diff
@@ -107,4 +129,18 @@ public StepResult doStep(FlightContext context) throws InterruptedException, Ret
 
     return StepResult.getStepResultSuccess();
   }
+
+  private String signGsUrl(Storage storage, String gsPath) {
```

Comment:
Yeah, I've been torn because there are a couple of different ways that we sign urls.  But yeah, that's fair.

Reply:
I created this simple signing method but I held off on migrating the other uses since those a bit more involved for now (have to deal with requester pays, etc.)